src/pal/src/thread/process.cpp

```diff
@@ -1983,6 +1990,22 @@ PAL_NotifyRuntimeStarted()
     return launched;
 }
 
+#ifdef __APPLE__
+LPCSTR
+PALAPI
+PAL_GetApplicationGroupId()
+{
+    return gApplicationGroupId;
+}
+
+BOOL
+PALAPI
+PAL_IsApplicationSandboxed()
```

Comment:
A nit - the PAL_IsApplicationSandboxed is not necessary, the caller can just check whether the return value of PAL_GetApplicationGroupId is NULL or not and we can document that. I'd like to keep the PAL surface as small as possible.